src/Services/ShopService/Order.php

```diff
@@ -36,5 +41,6 @@ public function __construct(array $data = [])
         $this->email = Arr::get($data, 'email');
         $this->discount = Arr::get($data, 'discount');
         $this->closed_at = Arr::get($data, 'closed_at');
+        $this->customer = Arr::get($data, 'customer');
```

Comment:
geef als default een new Customer object mee. Zodat customer altijd een Customer ook al is het een lege

Reply:
Dat doet me trouwens denken alle models moeten een fill method krijgen. Die eigenlijk het zelfde doet als hier in de constructor, maar zodat je op een later moment nog kan vullen bijv: $order->customer->fill($data);

Reply:
Helemaal in de bovenste map van onze Services wil ik een ServiceModelInterface met deze methods:
fill

Dan wil ik dat alle service models deze interface implements